transport/src/test/java/io/netty5/channel/nio/NioEventLoopTest.java

```diff
@@ -162,30 +163,44 @@ public void testSelectableChannel() throws Exception {
 
     @Test
     public void testTaskRemovalOnShutdownThrowsNoUnsupportedOperationException() throws Exception {
```

Comment:
tbh, I don't know what this test is trying to achieve. It seems that it's goal is to get the rejected-execution-exception. Can this be better done by stalling a task via a countdown latch?